src/Http/Requests/UpdateFormBuilderFormRequest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace FluxErp\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateFormBuilderFormRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            'id' => 'required|exists:form_builder_forms,id,deleted_at,NULL',
+            'model_type' => 'required_with:model_id|string',
+            'model_id' => 'required_with:model_type|integer',
```

Comment:
remove model morph from update request, I don't think this needs to be updateable

Reply:
done